sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -4059,6 +4059,15 @@ object SQLConf {
       .booleanConf
       .createWithDefault(true)
 
+  val LEGACY_KEEP_PARTITION_SPEC_AS_STRING_LITERAL =
```

Comment:
Can we put it near LEGACY_PARSE_NULL_PARTITION_SPEC_AS_STRING_LITERAL and follow its cofig doc as well?

Reply:
Yes.